rollup.config.js

```diff
@@ -0,0 +1,82 @@
+import typescript from "@rollup/plugin-typescript";
+import json from "@rollup/plugin-json";
+import shebang from "@walrus/rollup-plugin-shebang";
+import multiInput from "rollup-plugin-multi-input";
+import fs from "fs";
+import commonjs from "@rollup/plugin-commonjs";
+import resolve from "@rollup/plugin-node-resolve";
+import pkge from "./package.json";
+
+// Right now,  vercel/pkg  does not support ESM libraries (https://github.com/vercel/pkg/issues/782)
+// For example, some files rely on @govtechsg/open-attestation, @govtechsg/oa-XX. These libraries in turn rely on jsonLd library, which is an ESM library
```

Comment:
it's not true, it's not jsonld.

We can write that some transitive dependencies uses ESM and uses https://github.com/standard-things/esm which is not compatible with vercel/pkg